framework/common/servicedef/services.xml

```diff
@@ -45,6 +45,10 @@ under the License.
             location="org.apache.ofbiz.common.CommonServices" invoke="echoService">
         <description>Echos back all passed parameters</description>
     </service>
+    <service name="echo-service" engine="clojure" validate="false"
```

Comment:
I suggest to use camel case for the name, like 'echoClojureService'

Reply:
is this a hard rule? 
echo-service is closer to clojure semantics. 
Clojure can have functions called like: "are-we>there-yet?". 
If it's not a requirement, I would prefer to keep it as is. 


Also, IMO OFBiz could use namespaces for services - it will help in grouping them based on functionality and will keep the names shorter:

party/xxxx
order/xxx
accounting/xxxx

Reply:
Namespaces could also help in people providing similar services under their own plugin / name without conflicting with existing services.

Reply:
At the begining I would reponse that its a hard rule but after wound up the timeline I saw : 
 * a cleanning of bad service naming [1]
 * no information on it in OFBiz best pratice (only on databse) [2] , only on entity name
 * remember a discussion from Adrian Crum but I couldn't get my hands on it again

At this time all service follow this rule, so this question is interresting to what rule we need to follow, because the service engine is a layer to abstract the language so why follow a rule link to a language but in other way the service is dedicate to test a engine so linked to this language.

My feel would be follow the layered practice and invisible the language behind (exclude the service to test an engine :sweat_smile: ) .

Concerning the namespace, clearly is a subject but i'm not sure to involve it here with blur the main purpose. I suggest to share on this part after directly on dev list :)

[1] https://svn.apache.org/viewvc?view=revision&revision=1369948
[2] https://cwiki.apache.org/confluence/display/OFBIZ/Best+Practices+Guide

Reply:
Renamed. 
We can approach this issue at a later time. 
I'll make a proposal in the next future to make OFBiz composable and leaner.